package.json

```diff
@@ -13,6 +13,9 @@
         "lines": 100,
         "statements": -10
       }
+    },
+    "moduleNameMapper": {
```

Comment:
Awesome ty!! I changed it to alias so now the paths are avocado/some-dir/etc! but I found that I had to keep "moduleNameMapper" inside Jest, otherwise the tests would show: 

<img width="594" alt="Screen Shot 2019-08-29 at 3 46 52 PM" src="https://user-images.githubusercontent.com/50588415/63971499-9bc66b80-ca74-11e9-91ef-be8a9256e43b.png">